Source/PLCrashReport.proto

```diff
@@ -360,4 +360,15 @@ message CrashReport {
 
     /* Report format information. Required for all v1.1+ crash reports. */
     optional ReportInfo report_info = 9;
+
+    /*
+     * Report format information
+     */
+    message UserInfo {
+        /* Custom string set by user */
+        optional string data = 1;
+    }
+
+    /* Custom user information. Can be used by user to store contextual information for the crash. */
+    optional UserInfo user_info = 10;
```

Comment:
I have two suggestion here:
1. Store string directly to simplify structure.
2. Make it more generic like `custom_data` or so. Since it's up to user (of the lib) what to store there - it can be not *user* and cause some confusions

Reply:
Ok, simplified to a string with name `custom_data`